src/fu-util.c

```diff
@@ -1703,6 +1703,49 @@ fu_util_perhaps_refresh_remotes (FuUtilPrivate *priv, GError **error)
 	return fu_util_download_metadata (priv, error);
 }
 
+static gchar *
+fu_util_time_to_str (guint64 tmp)
+{
+	/* unknown, so should not be printed really */
+	if (tmp == 0) {
```

Comment:
Maybe this should actually be an assertion, this function is expected to be called with non-zero.

Reply:
Fixed, but I used a `g_return_val_if_fail()` rather than an assert.